app/views/games/show.html.erb

```diff
@@ -50,3 +69,55 @@
     </div>
   <% end %>
 </div>
+
+<div class="modal fade" id='myModal' role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true" data-keyboard="false" data-backdrop="static">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title" id="exampleModalLabel">Promote Your Pawn</h4>
+      </div>
+      <form action="#">
+        <div class="promo-selection-container">
```

Comment:
You're right, we could get rid of some of this repetition with something like (note I haven't actually run this code, so there might be some syntax errors, etc.): 

```ruby
<% promo_types = %w(Queen Knight Rook Bishop) %>
<% promo_types.each do |promotion| %> 
 <div class="checkboxgroup">
             <label for="<%= promotion %>.png" class="radio-label">
               <img src="/assets/white<%= promotion %>.png" id="<%= promotion.chars.first.downcase %>" class="piece-image">
               <br />
               <%= promotion %>
             </label>
             <input class="radio-input" type="radio" name="p_type" value="<%= promotion %>" checked="checked">
           </div>
<% end %>
```